data_steward/analytics/cdr_ops/controlled_tier_qc/check_controlled_tier_part2.py

```diff
@@ -518,47 +525,90 @@ def my_sql(table_name, column_name):
 
 # +
 query = JINJA_ENV.from_string("""
-WITH person_ehr as (
+WITH src_ids as (
+  SELECT 
+    src_id
+  FROM
+    `{{project_id}}.pipeline_tables.site_maskings`
+  WHERE NOT
+    REGEXP_CONTAINS(src_id, r'(?i)(PPI/PM)|(EHR site)')
+),
 
-SELECT distinct person_id FROM `{{project_id}}.{{ct_dataset}}.observation`
-JOIN `{{project_id}}.{{ct_dataset}}.observation_ext` USING (observation_id)
-WHERE   src_id !='PPI/PM'
```

Comment:
I think this had a different meaning in the past. The meaning would have been REGEXP_CONTAINS(src_id, r'(?i)EHR site') since the check is for ehr sites only. 

Suggestion: Remove (PPI/PM) from the src_ids CTE or remove the src_ids CTE entirely and go with a WHERE REGEXP_CONTAINS(src_id, r'(?i)EHR site') in each of the other CTEs.

What do you think?